<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;
</string>
  </property>
  <widget class="QPushButton" name="btnUserRegister">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#F5C26B;
border-radius:10px;}

QPushButton:pressed{background-color:#4169E1; color:white;
border-radius:20px; border-width:10px;}

QPushButton:hover{background-color:#F5C26B; border-width:10px;
border-radius:20px;}</string>
   </property>
   <property name="text">
    <string>반려동물 등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancle">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#F5C26B;
border-radius:10px;}

QPushButton:pressed{background-color:#4169E1; color:white;
border-radius:20px; border-width:10px;}

QPushButton:hover{background-color:#F5C26B; border-width:10px;
border-radius:20px;}</string>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelID">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBirthInfo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothicExtraBold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;</string>
   </property>
   <property name="text">
    <string>생일</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSpeciesInfo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothicExtraBold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>종</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameInfo_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothicExtraBold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>등록번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeightInfo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothicExtraBold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>몸무게</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelContactNumber">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="text">
    <string>010-0000-0000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelContactNumberInfo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothicExtraBold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>전화번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAgeInfo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothicExtraBold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;</string>
   </property>
   <property name="text">
    <string>나이</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAge">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameInfo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothicExtraBold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditBirth">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2024</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1990</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy. MM. dd.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editWeight">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSpecies">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="editContactNumber">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
